src/preface.md

```diff
@@ -73,7 +73,7 @@ myself. A lot of the initial code here is going to be similar to Phil’s.
 But I’m going to write about it anyway. There’s a good reason for that:
 
 > Writing is nature’s way of showing us how sloppy our thinking is.
-> 
+>
```

Comment:
This should just be a space and not a \n. I've yet to build the book. Will try it out to see